examples/all-clusters-app/esp32/README.md

```diff
@@ -160,73 +174,60 @@ your network configuration. To erase it, simply run.
 
           $ idf.py -p /dev/tty.SLAB_USBtoUART erase_flash
 
-### Setting up Python Controller
-
-Once ESP32 is up and running, we need to set up a device controller to perform
-commissioning and cluster control.
-
--   Set up python controller.
+#### Commission a device using chip-tool
 
-           $ cd {path-to-connectedhomeip}
-           $ ./scripts/build_python.sh -m platform
+To initiate a client commissioning request to a device, run the built executable
+and choose the pairing mode.
 
--   Execute the controller.
+#### Commission a device over BLE
 
-           $ source ./out/python_env/bin/activate
-           $ chip-device-ctrl
+Run the built executable and pass it the discriminator and pairing code of the
+remote device, as well as the network credentials to use.
 
-### Commissioning over BLE
+The command below uses the default values hard-coded into the debug versions of
+the ESP32 all-clusters-app to commission it onto a Wi-Fi network:
 
--   Establish the secure session over BLE. BLE is the default mode in the
-    application and is configurable through menuconfig.
+    $ chip-tool pairing ble-wifi ${NODE_ID_TO_ASSIGN} ${SSID} ${PASSWORD} 0 20202021 3840
```

Comment:
1. You have mentioned `out/debug/standalone/chip-tool` above and directly use `chip-tool` in the commands at all the places
2. The 0 in the above command is not required now

Reply:
resolved